LogoSvg.js

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
```

Comment:
Added import statement to give more options for nesting as well as styling in component return statement.